### PR TITLE
Made sure var 'pos' was only assigned locally to function

### PR DESCRIPTION
**Pull request by ZenonSeth**
The function is assigning to a var named `pos` but it was not declared as local, instead it was assigning to a global variable. Considering it's a really common name for a var, which messes with IDE linting, and that the fix is tiny, I think it's worth just making it `local`.

## To do

Ready for Review.

## How to test

n/a

### CODE REVIEW

**Comment by Desour** [comment]
We should use luacheck for our wireshark dissector, same as for any other lua code.

**Review by Desour — APPROVED** [review]

**Comment by appgurueu** [comment]
@Desour fwiw, trivial PRs require only one approval AFAIK. You could hit merge right away.

**Comment by Desour** [comment]
I know, but I don't want to do this immediately after I was the one to add the trivial tag.

**Pull request merged**
